glide-core/Cargo.toml

```diff
@@ -49,6 +49,7 @@ socket-layer = [
     "protobuf",
     "tokio-util",
 ]
+valkey-GTE-7-2 = []
```

Comment:
Please remove. When we build and release the client, it should work with all server versions. A user should be able to use the same client lib with all supported (= compatible) servers.

Reply:
Only version Valkey-7.2 and above support AZ affinity strategy, since only from this version the info command supports the availability-zone property, which is required to support this feature.

Reply:
We're working on client, it should support all server versions. When user runs an application with GLIDE, the client does not know the server version.

Reply:
It's a feature for tests. Users don't interact with the rust part as a crate/module/package/... for them, it's a binary comes with the binding they use, and the feature doesn't exist for them.
For us, to avoid running the tests on other versions, the feature is helpful. It's a bit different technically from the binding way to deal with it because we need to know what is the case before we're creating the connection. Meaning, we need to know if to create the client with or without the relevant config, and the current testing platform of redis-rs doesn't have anything similar in matter of versions.
When we improve the testing framework of redis-rs, which needs plenty of refactorings, this implementation can be reevaluated.
In any case, this is not relevant in any way to the end user, and the feature change something just in the testing area.

Reply:
So we can skip these tests on some server version(s) identified in runtime, as we do in all clients, can't we?

Reply:
Yes, but it needs extra functionality that doesn't exist.
I agree it's more aligned and clean to do as in all clients, but because of the timeline I would separate it to another task. @adarovadya Can open an issue and assign herself, and add the testing functionalities required after the freeze. It has no effect on the feature or on it being really tested.

Reply:
https://github.com/valkey-io/valkey-glide/issues/2688